docs/cmdline-opts/list-only.d

```diff
@@ -2,15 +2,15 @@ c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 Long: list-only
 Short: l
-Protocols: FTP POP3
+Protocols: FTP POP3 SFTP
 Help: List only mode
 Added: 4.0
-Category: ftp pop3
+Category: ftp pop3 sftp
 Example: --list-only ftp://example.com/dir/
 See-also: quote request
 Multi: boolean
 ---
-(FTP)
+(FTP SFTP)
```

Comment:
You can remove this line, it's superfluous and just adds duplicated info.

Reply:
I mean remove that line completely. It was wrong already before your change so you can fix that while there.

Reply:
OK, I hope I did it correctly.
I wonder if I should add a note about SFTP server listing . and .. as well. It this the same behavior as the FTP servers?

Reply:
What behavior is that you mean? FTP directory listings are not very standardized so they can vary a little between server implementations.